README.md

```diff
@@ -2,126 +2,55 @@
 
 ![Build and Test](https://github.com/Jigsaw-Code/outline-server/actions/workflows/build_and_test_debug.yml/badge.svg?branch=master) [![Mattermost](https://badgen.net/badge/Mattermost/Outline%20Community/blue)](https://community.internetfreedomfestival.org/community/channels/outline-community) [![Reddit](https://badgen.net/badge/Reddit/r%2Foutlinevpn/orange)](https://www.reddit.com/r/outlinevpn/)
 
-This repository has all the code needed to create and manage Outline servers on
-DigitalOcean. An Outline server runs instances of Shadowsocks proxies and
-provides an API used by the Outline Manager application.
+Outline Server is a tool to create and manage your own VPN servers using Shadowsocks proxies. If you prefer a graphical interface, you can use the [Outline Manager](https://github.com/Jigsaw-Code/outline-apps/).
```

Comment:
This characterization sounds odd to me. 

People don't really use the Outline Server as a "tool".
Also, you don't really use the server to manager servers.

Perhaps it makes sense to say something about the repository instead. Like, "This repository has the server components used by the Outline product"

Reply:
I mean the server is a tool to manage shadowsox instance? But I see your point, "tool" may imply more of a script.

I want to make sure this intro makes it clear what the "Outline Server" is in isolation from the rest of the product, for readers who stumble onto this repo specifically. "server components" implies you can't use the server without the other products, which is not correct. Especially if we want to support large-scale providers who won't use the Manager.

How about:

"Outline Server is the server component that manages Shadowsocks proxies and is used by the Outline product. You can deploy this server directly following simple instructions in this repository, or if you prefer a ready-to-use graphical interface you can use the Outline Manager."

Thoughts?

Reply:
Perhaps "Outline Server is the component that provides the Shadowsocks service (via outline-ss-server) and a service management API. You can deploy this server directly following simple instructions in this repository, or if you prefer a ready-to-use graphical interface you can use the Outline Manager."

I want to avoid the proxies plural, since that sounds like multiple proxies.